generators/root/templates/base/README.md

```diff
@@ -0,0 +1,27 @@
+# README
+
+## Prerequisites
+
+Docker and docker-compose must be installed and running.
+
+## Install
+
+Simply run the following script to initiate bootstrapping
+
+```
+./script/bootstrap
+```
+
+## Run
+
+Launch your containers by using :
+
+```
+./script/server
+```
+
+The project is available on localhost:80
+
+## Provision a new environment
+
+Check under `/environments` folder for Terraform information about provisionning
```

Comment:
This depends of your deployment mode, at least until https://github.com/thetribeio/generator-project/pull/1393 is finished so this should be templated to changed based on the chosen mode.